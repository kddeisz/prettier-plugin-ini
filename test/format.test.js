const fs = require("fs");
const path = require("path");
const prettier = require("prettier");

const fixture = fs.readFileSync(path.join(__dirname, "./fixture.ini"), "utf-8");
const format = content => prettier.format(content, { parser: "ini", plugins: ["."] });

test("defaults", () => {
  const formatted = format(fixture);
  expect(formatted).toMatchSnapshot();
});
