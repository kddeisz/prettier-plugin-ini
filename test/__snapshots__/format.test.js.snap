// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`defaults 1`] = `
"; last modified 1 April 2001 by John Doe

[owner]
name=John Doe
organization=Acme Widgets Inc.

[database]
; use IP address in case network name resolution is not working
server=192.0.2.62
port=143
file=\\"payroll.dat\\"
# this is another kind of comment
"
`;
